Update Mojo install command (#91)
mulesoft/docs-mule-sdk: modules/ROOT/pages/rest-sdk/project-setup.adoc

A _mojo_ is a Maven plain Old Java Object.
Each mojo is an executable goal in Maven, and a _plugin_ is a distribution of one or more related mojos.

Maven automatically downloads the plugin when you try to use it.

> You can run the following command to ensure you have access to necessary repositories and the latest plugin version:

Reviewer: ```suggestion
Run the following command to ensure you have access to necessary repositories and the latest plugin version:
```

* Where `<PATH_TO_SPEC>` is the path to the location of your API spec.
* Where `<PLUGIN_VERSION>` is the version of the Rest SDK Plugin you want to use.

> You can omit this parameter, and the latest version will be used.

Reviewer: ```suggestion
If you omit this parameter, the latest version is used.
```

* Where `<PATH_TO_SPEC>` is the location of your API spec.
* Where `<PATH_TO_DESCRIPTOR>` is the location of your descriptor.
* Where `<PLUGIN_VERSION>` is the version of the Rest SDK Plugin you want to use.

> You can omit this parameter, and the latest version will be used.

Reviewer: ```suggestion
If you omit this parameter, the latest version is used.
```
